kubernetes/samples/scripts/domain-lifecycle/helper.sh

```diff
@@ -0,0 +1,237 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+#
+# Function to create server start policy patch string
+# $1 - Domain resource in json format
+# $2 - Name of server whose policy will be patched
+# $3 - Policy value 
+# $4 - Return value containing server start policy patch string
+#
```

Comment:
Minor point: Looks like there's now a $5 - if you have the time, please make sure your function comments are up to date.

Reply:
Fixed comment.